ecommerce_integrations/shopify/order.py

```diff
@@ -207,7 +208,7 @@ def update_taxes_with_shipping_lines(taxes, shipping_lines, setting):
 				{
 					"charge_type": _("Actual"),
 					"account_head": get_tax_account_head(shipping_charge),
-					"description": shipping_charge["title"],
+					"description": get_tax_account_description(shipping_charge) or shipping_charge["title"],
```

Comment:
This is becoming a bit misleading now, anyway I'll refactor this later.

Reply:
All right, maybe this might be obsolete when using my other pullrequest...